Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Offset_meshing_plugin.cpp

```diff
@@ -390,27 +390,74 @@ class Polyhedron_demo_offset_meshing_plugin :
       connect(actionOffsetMeshing, SIGNAL(triggered()),
               this, SLOT(offset_meshing()));
     }
+
+    actionInflateMesh= new QAction(tr("Inflate Mesh"), mw);
+    actionInflateMesh->setProperty("subMenuName", "Operations on Polyhedra");
+    if(actionInflateMesh) {
+      connect(actionInflateMesh, SIGNAL(triggered()),
+              this, SLOT(inflate_mesh()));
+    }
   }
 
-  bool applicable(QAction*) const {
+  bool applicable(QAction* a) const {
     Scene_item* item = scene->item(scene->mainSelectionIndex());
+
     return
-      qobject_cast<Scene_surface_mesh_item*>(item) ||
-      qobject_cast<Scene_polygon_soup_item*>(item);
+        qobject_cast<Scene_surface_mesh_item*>(item) ||
+        (a == actionOffsetMeshing && qobject_cast<Scene_polygon_soup_item*>(item));
   }
 
   QList<QAction*> actions() const {
-    return QList<QAction*>() << actionOffsetMeshing;
+    return QList<QAction*>() << actionOffsetMeshing
+                             << actionInflateMesh;
   }
 public Q_SLOTS:
   void offset_meshing();
+  void inflate_mesh();
 
 private:
   QAction* actionOffsetMeshing;
+  QAction* actionInflateMesh;
   Scene_interface *scene;
   QMainWindow *mw;
 }; // end class Polyhedron_demo_offset_meshing_plugin
 
+void Polyhedron_demo_offset_meshing_plugin::inflate_mesh()
+{
+  const CGAL::Three::Scene_interface::Item_id index = scene->mainSelectionIndex();
+  Scene_item* item = scene->item(index);
+  Scene_surface_mesh_item* sm_item =
+      qobject_cast<Scene_surface_mesh_item*>(item);
+
+  SMesh* sMesh = sm_item->face_graph();
+  if(!sMesh)
+    return;
+  Scene_item::Bbox box = bbox(sMesh);
+
+  double X=(box.max)(0)-(box.min)(0),
+      Y = (box.max)(1)-(box.min)(1),
+      Z = (box.max)(2)-(box.min)(2);
+  double diag = sm_item->diagonalBbox();
+  double offset_value = QInputDialog::getDouble(mw,
+                                                QString("Choose Inflate Value"),
+                                                QString("Offset Value (use negative number for deflate)"),
```

Comment:
```suggestion
                                                QString("Offset Distance (use negative number for deflate)"),
```
because is a bit vague: it could have been a ratio of something. With "distance" the user will know what it is.